rootfs/scheduler/resources/pod.py

```diff
@@ -719,7 +719,41 @@ def _handle_not_ready_pods(self, namespace, labels):
                     message = "\n".join([x.strip() for x in event['message'].split("\n")])
                     raise KubeException(message)
 
-        return None
+        # Handle cases when no one pods is started (quota check fialed)
+        # If quota is enabled in a namespace for compute resources like cpu and
+        # memory, users must specify requests or limits for those values;
+        # otherwise, the quota system may reject pod creation.
+        # http://kubernetes.io/docs/admin/resourcequota/
+        # or when scale process trigger overuse case for some pods:
+        # For example: used 3/3 pods, but user scale to 4
+        events = self._get_namespace_events(namespace)
+        if len(events) != 0:
+            log = self._get_formatted_messages(events)
+            self.log(namespace, log)
+            raise KubeException(log)
```

Comment:
As described in kubernetes controller:
https://github.com/kubernetes/kubernetes/blob/f18a921a0308c4ecd2e7e8b7e30e39a8216a3447/pkg/controller/deployment/util/deployment_util.go#L612

To get new RS need to get template by deployment and match RS to deployment template. And i have no idea, how to get last RS for specified deployment in deis controller. So, we can't detect object which cause error and what error.

And I actually don't understand why we need to return None when required pods != actual pods during operation.

I can add regexp to detect patterns with "exceeded quota", but this message may changes in next releases.